apps/dashboard/src/app/knowledge-box/knowledge-box-process/knowledge-box-processes.component.html

```diff
@@ -23,10 +23,18 @@ <h2>{{ 'stash.settings.processes.' + type | translate }}</h2>
       <pa-dropdown #dropdown
                    (onOpen)="open[type] = true"
                    (onClose)="open[type] = false">
+        <pa-option class="toggle-all"
+                   *ngIf="labelsets | async as labelsets"
+                   dontCloseOnSelect>
+          <pa-checkbox [value]="selectedLabelsets[type].length === labelsets.length"
+                       (change)="toggleAll(type)">
+            {{ 'stash.settings.processes.training.all_labelsets' | translate }}
+          </pa-checkbox>
+        </pa-option>
```

Comment:
you can add a `<pa-separator></pa-separator>` below your option to add the line you see in the design:
<img width="749" alt="image" src="https://user-images.githubusercontent.com/3293937/182886071-2df953e8-f129-450e-b2bb-c04d67abc934.png">

Reply:
Fixed 👍